template/parse.go

```diff
@@ -312,6 +314,11 @@ func Parse(r io.Reader) (*Template, error) {
 	return rawTpl.Template()
 }
 
+// Parse takes the given io.Reader and parses a Template object out of it without attaching any hooks.
+func Parse(r io.Reader) (*Template, error) {
+	return parse(r, nil)
```

Comment:
passing `nil` as hook causes `Parse()` to return different output than `ParseFile()` which might be unexpected